mylyn.tasks/connectors/gitlab/org.eclipse.mylyn.gitlab.core.test/src/org/eclipse/mylyn/gitlab/core/test/GitlabTestFixture.java

```diff
@@ -0,0 +1,126 @@
+/*******************************************************************************
+ * Copyright © 2024 frank
```

Comment:
please add more details

Reply:
I'll change the headers.
Oddly enough, I found another place where I had a © instead of "(c)", so I'll change that too.

Reply:
My fault. I'll update the 'New Class' template